diff -rupN siesta-3.2-pl-5.orig/Obj/arch.make siesta-3.2-pl-5/Obj/arch.make
--- siesta-3.2-pl-5.orig/Obj/arch.make	1970-01-01 12:00:00.000000000 +1200
+++ siesta-3.2-pl-5/Obj/arch.make	2014-09-24 15:49:00.920474000 +1200
@@ -0,0 +1,67 @@
+# 
+# This file is part of the SIESTA package.
+#
+# Copyright (c) Fundacion General Universidad Autonoma de Madrid:
+# E.Artacho, J.Gale, A.Garcia, J.Junquera, P.Ordejon, D.Sanchez-Portal
+# and J.M.Soler, 1996- .
+# 
+# Use of this software constitutes agreement with the full conditions
+# given in the SIESTA license, as signed by all legitimate users.
+#
+.SUFFIXES:
+.SUFFIXES: .f .F .o .a .f90 .F90
+
+SIESTA_ARCH=EASYBUILD
+
+FPP=$EBVARF90
+FPP_OUTPUT= 
+FC=$EBVARF90
+RANLIB=ranlib
+
+SYS=nag
+
+SP_KIND=4
+DP_KIND=8
+KINDS=$(SP_KIND) $(DP_KIND)
+
+FFLAGS=$EBVARFFLAGS
+FPPFLAGS= -DMPI -DFC_HAVE_FLUSH -DFC_HAVE_ABORT
+LDFLAGS=-I$(EBVARBLACS_INC_DIR) -I$(EBVARFFTW_INC_DIR) -I$(EBVARMPI_INC_DIR) -static-intel
+
+ARFLAGS_EXTRA=
+
+FCFLAGS_fixed_f=
+FCFLAGS_free_f90=
+FPPFLAGS_fixed_F=
+FPPFLAGS_free_F90=
+
+BLAS_LIBS=-L$(EBVARBLAS_LIB_DIR) -I$(EBVARBLAS_INC_DIR) -I$(EBVARLAPACK_INC_DIR)/intel64/lp64 -lmkl_blas95_lp64
+LAPACK_LIBS=-L$(EBVARLAPACK_LIB_DIR) -I$(EBVARLAPACK_INC_DIR) -I$(EBVARLAPACK_INC_DIR)/intel64/lp64 -lmkl_lapack95_lp64
+BLACS_LIBS=
+SCALAPACK_LIBS=
+
+COMP_LIBS=
+
+NETCDF_LIBS=
+NETCDF_INTERFACE=
+
+LIBS=$(SCALAPACK_LIBS) $(BLACS_LIBS) $(LAPACK_LIBS) $(BLAS_LIBS) $(NETCDF_LIBS)
+
+#SIESTA needs an F90 interface to MPI
+#This will give you SIESTA's own implementation
+#If your compiler vendor offers an alternative, you may change
+#to it here.
+MPI_INTERFACE=
+MPI_INCLUDE=
+
+#Dependency rules are created by autoconf according to whether
+#discrete preprocessing is necessary or not.
+.F.o:
+	$(FC) -c $(FFLAGS) $(INCFLAGS) $(FPPFLAGS) $(FPPFLAGS_fixed_F)  $< 
+.F90.o:
+	$(FC) -c $(FFLAGS) $(INCFLAGS) $(FPPFLAGS) $(FPPFLAGS_free_F90) $< 
+.f.o:
+	$(FC) -c $(FFLAGS) $(INCFLAGS) $(FCFLAGS_fixed_f)  $<
+.f90.o:
+	$(FC) -c $(FFLAGS) $(INCFLAGS) $(FCFLAGS_free_f90)  $<
+
